public/app/plugins/datasource/grafana/types.ts

```diff
@@ -0,0 +1,34 @@
+import { AnnotationQuery, DataQuery, SelectableValue } from '@grafana/data';
+
+//----------------------------------------------
+// Query
+//----------------------------------------------
+
+export enum GrafanaQueryType {
+  RandomWalk = 'randomWalk',
+  RandomStream = 'randomStream',
+  HostMetrics = 'hostmetrics',
```

Comment:
Not implemented yet... could remove now, but also not used or referenced :shrug: